opm/parser/eclipse/EclipseState/SummaryConfig/tests/SummaryConfigTests.cpp

```diff
@@ -55,8 +55,9 @@ static DeckPtr createDeck( const std::string& summary ) {
             "/\n"
             "COMPDAT\n"
             "'PRODUCER'   5  5  1  1 'OPEN' 1* -1  0.5  / \n"
-            "'W_1'   3    7    1    3      'OPEN'  1*     32.948      0.311   3047.839  2*         'X'     22.100 / \n"
             "'W_1'   3    7    2    2      'OPEN'  1*          *      0.311   4332.346  2*         'X'     22.123 / \n"
```

Comment:
I mean here. I just realized that you deleted the above line while this line is not added by you. I misread it. Sorry about that. 

`*` indicating defaulted values are not supported by Eclipse, while we have so many this kinds of examples in our data decks in the tests.